### PR TITLE
Fixed wiki manuals, added wiki links for antags.

### LINKED ISSUE #2022: Hacking Book Leads to 404 Error
http://i.imgur.com/jvETOOu.png?1

What is the page it's suppose to link to?

### PR DESCRIPTION
**Pull request by Zadeon**
This fixes #2022 and adds links to most antag greeting messages for new players to use.

### CODE REVIEW

**Comment by Spacedong** [comment]
very good :+1:

**Comment by TheOneAndOnlyCreeperJoe** [comment]
Oh lovely, another good reason why I should work on antag pages on the wiki instead of anything else.

But uhh, pretty nice :+1:

**Comment by Carbonhell** [comment]
code's ok,waiting for headmin approval to merge

**Comment by myontasm** [comment]
:+1: Gucci

**Comment by Nebyab** [comment]
ok

**Comment by nalarmg** [comment]
Possibly worth making each code-used page a configuration option so updating them doesn't require a coder if something changes.

**Comment by Carbonhell** [comment]
It's already like that, nalar, through different wikis have different formats,so that can't be completely done. But it is compatible with mediawiki-based wikis like ours

File: code/game/gamemodes/shadowling/shadowling.dm [hunk 1]
```diff
@@ -116,8 +116,7 @@ Made by Xhuis
 	shadow.current << "<b>Currently, you are disguised as an employee aboard [station_name()]].</b>"
 	shadow.current << "<b>In your limited state, you have three abilities: Enthrall, Hatch, and Hivemind Commune.</b>"
 	shadow.current << "<b>Any other shadowlings are your allies. You must assist them as they shall assist you.</b>"
-	shadow.current << "<b>If you are new to shadowling, or want to read about abilities, check the wiki page at https://tgstation13.org/wiki/Shadowling</b><br>"
-
+	shadow.current << "<b>If you are new to shadowling, or want to read about abilities, check the wiki page at http://wiki.hippiestation.com/index.php?title=Shadowling</b><br>"
```
Comment (danhibiki) [inline code/game/gamemodes/shadowling/shadowling.dm]: You missed one

**Comment by Nokington** [comment]
k

**Comment by Pyko1** [comment]
enough approvals and apparently code is fine, merging

**Pull request merged**